src/components/learner-credit-management/cards/NewAssignmentModalButton.jsx

```diff
@@ -41,9 +44,25 @@ const NewAssignmentModalButton = ({ course, children }) => {
 
   const pathToActivityTab = generatePath(routeMatch.path, { budgetId: subsidyAccessPolicyId, activeTabKey: 'activity' });
 
+  const handleOpenAssignmentModal = () => {
+    open();
+    sendEnterpriseTrackEvent(
+      enterpriseId,
+      EVENT_NAMES.LEARNER_CREDIT_MANAGEMENT.OPEN_ASSIGNMENT_MODAL_ASSIGN_COURSE,
+      {
+        isOpen: !isOpen,
```

Comment:
[curious] I'm wondering if the `isOpen` property is largely derived from the event name? For example, would the event name `EVENT_NAMES.LEARNER_CREDIT_MANAGEMENT.OPEN_ASSIGNMENT_MODAL_ASSIGN_COURSE` always indicate that the modal was opened?

Similar comment for the `handleCloseAssignmentModal` below, too, wrt to the `isOpen` property.

Reply:
It might be worth removing the OPEN/CLOSE prefixes on segments event related to this and have the event metadata speak for itself whether it isOpen or !isOpen 😺